### PR TITLE
Multilingual support [TESTING]

### PR DESCRIPTION
**Pull request by OfficialNoob**
Using the native API add multiple language support.

### CODE REVIEW

**Comment by Jackymancs4** [comment]
Sorry to bother you once more, but (while I'm not sure what are you doing) this seems not what you want.

If you're trying to localize the extension you definetely should use the Localizations API provided both by Chrome and Firefox

For documentation:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization

**Comment by OfficialNoob** [comment]
@Jackymancs4 sure I will use the native API

File: LANG.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.set("CWL","Channel Whitelisted!")
+.set("SPs","sponsors.")
+.set("SP","sponsor.")
+.set("SPSEGs","sponsor segments.")
+.set("SPSEG","sponsor segment.")
+.set("SP_FOUND","This video's sponsors are in the database!")
+.set("SP_NONE","No sponsors found")
+.set("SP_END","Sponsorship Ends Now")
+.set("SP_START","Sponsorship Starts Now")
+.set("NOTYT","his probably isn't a YouTube tab, or you clicked too early. \n If you know this is a YouTube tab,\n close this popup and open it again.")
```
Comment (ajayyy) [inline LANG.js]: Missing a t in `his probably isn't`

Comment (OfficialNoob) [reply LANG.js]: Fixed

**Comment by OfficialNoob** [comment]
@ajayyy please review the current changes.

File: content.js [hunk 2]
```diff
@@ -578,7 +578,7 @@ function changeStartSponsorButton(showStartSponsor, uploadButtonVisible) {
   if (showStartSponsor) {
     showingStartSponsor = true;
     document.getElementById("startSponsorImage").src = chrome.extension.getURL("icons/PlayerStartIconSponsorBlocker256px.png");
-    document.getElementById("startSponsorButton").setAttribute("title", "Sponsor Starts Now");
+    document.getElementById("startSponsorButton").setAttribute("title", browser.i18n.getMessage("SponsorStart"));
```
Comment (ajayyy) [inline content.js]: We are using chrome.* everywhere, so I think we should stick with one to be consistent.

Comment (OfficialNoob) [reply content.js]: What do you mean?

Comment (ajayyy) [reply content.js]: Instead of `browser.*`

Comment (OfficialNoob) [reply content.js]: Ah ok will change

Comment (OfficialNoob) [reply content.js]: Changed to chrome

**Comment by OfficialNoob** [comment]
I might separate the help page per lang

**Comment by OfficialNoob** [comment]
@ajayyy Done :D

**Comment by ajayyy** [comment]
Does this apply to popup html not changed by the JS?

**Comment by OfficialNoob** [comment]
Yes popup.html is now popup_en.html

File: _locales/en/messages.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  "SponsorEND": {
+    "message": "Sponsorship Ends Now"
+  },
+  "noVIDEOID": {
+    "message": "This probably isn't a YouTube tab, or you clicked too early. \n If you know this is a YouTube tab,\n close this popup and open it again."
+  },
+  "VOTED": {
+    "message": "Thanks for voting!"
+  },
+  "VOTE_FAIL": {
```
Comment (ajayyy) [inline _locales/en/messages.json]: Why is some stuff all caps, some stuff `PascalCase` and some stuff `camelCase`?

File: manifest.json [hunk 4]
```diff
@@ -41,8 +42,8 @@
     "https://sponsor.ajay.app/*"
   ],
   "browser_action": {
-    "default_title": "SponsorBlock",
-    "default_popup": "popup.html"
+    "default_title": "__MSG_Name__",
+    "default_popup": "__MSG_popup__"
```
Comment (ajayyy) [inline manifest.json]: Where is this defined?

File: _locales/en/messages.json (hunk 3 above)
Comment (OfficialNoob) [reply _locales/en/messages.json]: Because theres no linter to stop me xD

File: manifest.json (hunk 4 above)
Comment (OfficialNoob) [reply manifest.json]: https://github.com/OfficialNoob/SponsorBlock/blob/patch-18/_locales/en/messages.json

Comment (ajayyy) [reply manifest.json]: Oh, does `__MSG_popup_` just refer to `popup`?

Comment (OfficialNoob) [reply manifest.json]: yes

File: _locales/en/messages.json (hunk 3 above)
Comment (ajayyy) [reply _locales/en/messages.json]: You are purposefully changing format randomly?

Comment (OfficialNoob) [reply _locales/en/messages.json]: no there was no rules in place for json files

Comment (OfficialNoob) [reply _locales/en/messages.json]: Nope its changed now

**Comment by OfficialNoob** [comment]
Made changes to comply with naming convention + added more into locales

**Comment by ajayyy** [comment]
Does this fallback to english if no language file is found?

**Comment by ajayyy** [comment]
https://stackoverflow.com/a/39810769/1985387

**Comment by OfficialNoob** [comment]
What do you mean both pages get redirected eg index_en.html, popup_en.html

**Comment by ajayyy** [comment]
About [this comment](https://github.com/ajayyy/SponsorBlock/pull/86#issuecomment-520676810)?

**Comment by OfficialNoob** [comment]
@ajayyy its ready :D

**Comment by ajayyy** [comment]
I can't get this to work without manually specifying `popup-en.html`

**Pull request merged**